docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -2203,6 +2203,44 @@ will also be applied to `initContainers`.
   </TabItem>
 </Tabs>
 
+## `initSecurityContext`
+
+[Container security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)
+to set for the [initContainer](#initcontainers).
+Defaults to:
+
+```yaml
+initSecurityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - all
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 9807
+```
+
+To unset the security context, set it to `null` or `~`.
```

Comment:
What does `~` mean here?

Reply:
It's a fancy way of unsetting a field in yaml. Like `null`, but different of `{}`.

Reply:
TIL :pray: 
```
null | Null | NULL | ~
```

https://yaml.org/spec/1.2.2/